cmd/wallet-sdk-js/test/openid4ci.js

```diff
@@ -0,0 +1,142 @@
+import '../src/wasm_exec.js';
+import Agent from '../src/index.js';
+
+
+describe("A suite is just a function", function () {
+
+    it("and so is a spec", async function () {
```

Comment:
These should be probably describe what is being tested

Reply:
rewrited